src/pages/model/stackview/ModelPageStackViewContent.svelte

```diff
@@ -25,27 +25,36 @@
   function onUserInteraction(): void {
     hasInteracted = true;
   }
+
+  $: prediction = $bestPrediction
+    ? {
+        name: $bestPrediction.name,
+        certainty: `${Math.round($bestPrediction.confidence.currentConfidence * 100)}%`,
+      }
+    : {
+        name: $t('content.model.output.estimatedGesture.none'),
+      };
 </script>
 
 <h1 class="sr-only">{$t('content.index.toolProcessCards.model.title')}</h1>
 <div class="flex flex-col h-full bg-backgrounddark">
   <div class="flex justify-center space-x-10 py-5 text-xl border-b-3 border-b-gray-200">
+    <span class="sr-only" aria-live="polite" aria-atomic="true"
+      >{$t('content.model.output.estimatedGesture.iconTitle')}
+      {prediction.name}
+      {prediction.certainty ? prediction.certainty : ''}</span>
```

Comment:
Added `aria-atomic="true"` so that the whole thing would be read out, but not sure if that's a good thing for context, or whether it would just be annoying. Suggestions welcomed

Reply:
I think we need to change this so that the estimated action is only read out when the estimated action changes. At the moment it reads it every time the confidence/percentage changes so it can't keep up and gets very out of sync with what's actually happening. I think maybe we drop the percentage as it's rarely going to be accurate by the time it's read out.

Reply:
That makes sense, have removed the certainty part (https://github.com/microbit-foundation/ml-trainer/pull/149/commits/f8d49015c627f73ab60277efbc105c842361b037)